src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Chewy&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      href="https://ddo0fzhfvians.cloudfront.net/uploads/icons/png/611885851581482968-512.png"
+    />
+    <title>THE CHICKENEER</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="navbar-inner">
+        <span class="navbar-header">THE CHICKENEER</span>
+      </div>
+    </nav>
+    <div class="tabs">
+      <div class="home-tab">
+        <span>HOME</span>
+      </div>
+      <div class="menu-tab">
+        <span>MENU</span>
+      </div>
+      <div class="contacts-tab">
+        <span>CONTACT US</span>
+      </div>
+    </div>
+
+    <main id="content"></main>
+
+    <footer class="footer">
+      <div>
+        <span>
+          Made by
+          <a target="_blank" href=" https://github.com/mahmoud717">
+            Mahmoud Mohammad</a
+          >
+        </span>
+      </div>
+    </footer>

```

Comment:
_**You have really done well 👍**_ 

**Seems you missed the DOM instruction 🤔** 

According to the project instructions on the Odin page, we need to create the DOM element with JavaScript.
![image](https://user-images.githubusercontent.com/31897434/100893220-68fef480-34bb-11eb-9216-aeebcba83ddd.png)

And also, there is a heads-up about having just a div tag in the body tag;
![image](https://user-images.githubusercontent.com/31897434/100893559-c5621400-34bb-11eb-9676-d9d37fe6d785.png)